NumberValidator/Validators/IN/PAN.cs

```diff
@@ -20,22 +19,25 @@ public bool IsValid(string pan)
                 return false;
             }
         }
+
         public void Validate(string pan)
         {
-            string panPattern = @"^[A-Z]{5}\d{4}[A-Z]$";
+            pan = pan.RemoveSpace();
+
             if (pan.Length != 10)
             {
                 throw new InvalidLengthException();
             }
-            if (pan.Substring(4, 4) == "0000") // Change index from 5 to 4
+
+            if (pan.Substring(5, 4) == "0000")
             {
                 throw new InvalidComponentException();
             }
-            Regex panRegex = new Regex(panPattern);
-            if (!panRegex.IsMatch(pan))
+
+            if (!Regex.IsMatch(pan, @"^[A-Z]{5}[0-9]{4}[A-Z]$"))
```

Comment:
You guys said you collaborated. But both of you are producing duplicate changes in PAN.

Reply:
both logics are eventually same so i will remove it